bids/layout/models.py

```diff
@@ -1,18 +1,14 @@
 """ Model classes used in BIDSLayouts. """
 
-import re
 import os
 from pathlib import Path
-import warnings
 import json
-from copy import deepcopy
-from itertools import chain
-from functools import lru_cache
+from ancpbids.model_v1_8_0 import Artifact
```

Comment:
this is potentially risky as Artifact is bound to model_v1_8_0

Reply:
@erdalkaraca Could there be an abstract class `ancpbids.types.Artifact` that `ancpbids.model_*.Artifact` will inherit from? Or even a [Protocol](https://docs.python.org/3/library/typing.html#typing.Protocol)?